vscode-extension/PSA_custom_Rules/Alias/Get-AliasSpec.psm1

```diff
@@ -0,0 +1,16 @@
+#load the alias from the json file
+
+function Get-AliasSpec {
```

Comment:
This file is not a module, so it should be a `.ps1` not `.psm1`. Also please add "todo: pull the spec file online" here

Reply:
I don't know why this is not a module. In my idea, it's a module and I have import it in avoidAlias.psm1

Reply:
A module should use `Export-Member` to export what you want to expose. `Import-Module` should be used to import a module. Without them, this is just a plain PowerShell script that is executed in another script.